sdk/include/opentelemetry/sdk/trace/tracer.h

```diff
@@ -103,8 +105,10 @@ class Tracer final : public opentelemetry::trace::Tracer,
 private:
   // order of declaration is important here - instrumentation scope should destroy after
   // tracer-context.
+  std::unique_ptr<TracerConfig> tracer_config_;
```

Comment:
Replace with:
```
TracerConfig tracer_config_;
```

There is no need to allocate memory and use a unique pointer, the tracer can own it's own configuration directly.